src/Http/Session/DatabaseSession.php

```diff
@@ -180,10 +180,10 @@ public function destroy(string $id): bool
     /**
      * Helper function called on gc for database sessions.
      *
-     * @param int $maxlifetime Sessions that have not updated for the last maxlifetime seconds will be removed.
+     * @param int $max_lifetime Sessions that have not updated for the last maxlifetime seconds will be removed.
      * @return int|false The number of deleted sessions on success, or false on failure.
      */
-    public function gc(int $maxlifetime): int|false
+    public function gc(int $max_lifetime): int|false
```

Comment:
This should be ok since there is just one parameter and people wouldn't be using names.

Reply:
Yup, users don't need to call this method directly, so shouldn't be a problem.

Reply:
oh, true.